### ISSUE TITLE
NoMethodError: undefined method `constantize' for Resque:Module

### ISSUE THREAD

**Issue by jeremywadsack** [frausto/resque-remora#2]
Trying to use `resque-director` that depends on this, I get the following error. 

```
NoMethodError: undefined method `constantize' for Resque:Module
    from ~/.rvm/gems/ruby-2.1.0@keylime-toolbox/gems/resque-remora-0.1.0/lib/resque/plugins/remora/push_pop.rb:15:in `push'
    from ~/.rvm/gems/ruby-2.1.0@keylime-toolbox/gems/resque-1.25.2/lib/resque/job.rb:153:in `create'
```

I'm running resque 1.25.2, which is the latest on the 1.x-stable branch. Also much, much newer than the last commit to this repo. I see that there is a fork to patch this: https://github.com/wearemolecule/resque-remora/commit/8d73ce9f11adda7b3d43ee881a1d844fb146b35e

Any chance of getting this patched in a released gem?

**Comment by dreeve** [comment]
[This code](https://github.com/frausto/resque-remora/blob/master/lib/resque/plugins/remora/push_pop.rb#L40-L68) isn't included in the current version of this gem on rubygems. 

Would it be possible to release a version 0.2 that includes the changes here on github?

**Comment by raorao** [comment]
This should now be resolves as of version 0.2
